packages/adminrouter/docker/adminrouter-upstreams-ee.conf

```diff
@@ -1,17 +1,14 @@
 set $upstream_networking_api http://127.0.0.1:61430;
 
-proxy_ssl_trusted_certificate /run/dcos/pki/CA/certs/ca.crt;
+proxy_ssl_trusted_certificate /run/dcos/pki/CA/ca-bundle.crt;
 proxy_ssl_verify on;
 proxy_ssl_verify_depth 5;
 
-# For the sake of tests simplicity, we will not be using/testing leader.mesos
-# alias here. This should be done by integration tests.
-set $upstream_mesos http://127.0.0.2:5050;
-set $upstream_marathon http://127.0.0.1:8080;
+set $upstream_mesos http://leader.mesos:5050;
 set $upstream_secrets http://127.0.0.1:1337;
 set $upstream_cosmos http://127.0.0.1:7070;
 
 # Required by EE Agent:
-set $upstream_iam http://127.0.0.1:8101;
+set $upstream_iam http://master.mesos:8101;
```

Comment:
We do - in production. 

Test harness cheats a little by pushing requests directly to IAM mock, instead of starting two AR instances (Master + Agent). This helps to keep things simpler without any negative effect on tests (we are testing all IAM endpoints in Master using different/separate tests) :)